From: Paul Martin <pm@debian.org>
Date: Wed, 30 Aug 2017 16:24:03 +0200
Subject: Fix broken test suite runner

    That line appearently got lost between 3.8.7 and 3.11.0 but is
    required to make the tests run.
---
 test/test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/test b/test/test
index 1659887..09cf267 100755
--- a/test/test
+++ b/test/test
@@ -3,6 +3,7 @@
 ACL_TESTS=1
 SELINUX_TESTS=0
 SYSLOG_TESTS=0
+LOGROTATE=../logrotate
 M="-m ./mailer"
 S="-s state"
 RLR="$LOGROTATE $M $S"
